hack/README.md

```diff
@@ -177,22 +184,59 @@ not supported, then you can use the following example to run Functional Tests.
+   ##### Direct deployment
    ```
-   # kubectl create -f manifests/generated/cdi-controller.yaml
-
-     serviceaccount/cdi-sa created
+     #kubectl create -f ./_out/manifests/cdi-controller.yaml
+     
+     namespace/cdi created
+     customresourcedefinition.apiextensions.k8s.io/datavolumes.cdi.kubevirt.io created
+     customresourcedefinition.apiextensions.k8s.io/cdiconfigs.cdi.kubevirt.io created
      clusterrole.rbac.authorization.k8s.io/cdi created
      clusterrolebinding.rbac.authorization.k8s.io/cdi-sa created
+     clusterrole.rbac.authorization.k8s.io/cdi-apiserver created
+     clusterrolebinding.rbac.authorization.k8s.io/cdi-apiserver created
+     clusterrolebinding.rbac.authorization.k8s.io/cdi-apiserver-auth-delegator created
+     serviceaccount/cdi-sa created
      deployment.apps/cdi-deployment created
-     customresourcedefinition.apiextensions.k8s.io/datavolumes.cdi.kubevirt.io created
+     configmap/cdi-insecure-registries created
+     serviceaccount/cdi-apiserver created
+     rolebinding.rbac.authorization.k8s.io/cdi-apiserver created
+     role.rbac.authorization.k8s.io/cdi-apiserver created
+     rolebinding.rbac.authorization.k8s.io/cdi-extension-apiserver-authentication created
+     role.rbac.authorization.k8s.io/cdi-extension-apiserver-authentication created
+     service/cdi-api created
+     deployment.apps/cdi-apiserver created
+     service/cdi-uploadproxy created
+     deployment.apps/cdi-uploadproxy created
+
+   ```
+   ##### Deployment via operator
+   ```
+     #./cluster/kubectl.sh apply -f "./_out/manifests/release/cdi-operator.yaml" 
+     namespace/cdi created
+     customresourcedefinition.apiextensions.k8s.io/cdis.cdi.kubevirt.io created
+     configmap/cdi-operator-leader-election-helper created
+     clusterrole.rbac.authorization.k8s.io/cdi.kubevirt.io:operator created
+     serviceaccount/cdi-operator created
+     clusterrole.rbac.authorization.k8s.io/cdi-operator-cluster-permissions created
+     clusterrolebinding.rbac.authorization.k8s.io/cdi-operator created
+     deployment.apps/cdi-operator created
+
+     #./cluster/kubectl.sh apply -f "./_out/manifests/release/cdi-operator-cr.yaml"
+     cdi.cdi.kubevirt.io/cdi created
+
    ```
 
 4. Build and run the func test servers
    In order to run fucntional tests the below servers have to be run
    - *host-file-server* is required by the functional tests and provides an
      endpoint server for image files and s3 buckets
-   - *registry-server* is required by the functional tests and provides an endpoint server for container images.
+   - *registry-server* is required by the functional tests and provides an endpoint server for container images. 
+     Note: for this server to run the follwoing setting is required in each cluster node 
+     ``` systemctl -w user.max_user_namespaces=1024 ```
+   - *block-device* is required by functional tests that verify block dv functionality. 
```

Comment:
This is true, but if the base system does not have block devices, we actually run some pods to create some minimal test block devices. So I don't think we need to add this particular piece of text.

Reply:
removed